src/MakersOfDenmark.Domain/Models/MakerSpace.cs

```diff
@@ -16,6 +18,15 @@ public class MakerSpace : Entity<Guid>
         public AccessType AccessType { get; set; }
         public Organization Organization { get; set; }
         public ICollection<Tool> Tools { get; set; } = new List<Tool>();
-
+        public ICollection<MODUserFollowMakerSpace> Followers { get; set; } = new List<MODUserFollowMakerSpace>();
```

Comment:
Jeg tænker faktisk at der skal være en private set her. På den måde sørger vi for at tilføjelse/fjernelse styres af klassen.

Reply:
Forstyrrer det ikke ef?

Reply:
Nej. Det gør det ikke.